db/c.cc

```diff
@@ -1,4 +1,4 @@
-//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+///  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
meaningful or just a typo?

Reply:
typo.. I meant to perform a search in Emacs and missed the control key and instead typed an extra char I failed to delete.  will remove.